### PR TITLE
Fix bug in find command

### LINKED ISSUE #127: Update find feature
To match TreeMap implementation of a person’s fields.

### PR DESCRIPTION
**Pull request by tonishka**
Closes #127

### CODE REVIEW

File: src/main/java/seedu/address/model/person/FieldContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -58,7 +55,7 @@ private boolean testName(Person person) {
 
     private boolean testJob(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getJobTitle().toString(), keyword));
+                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getJobTitle().get().jobTitle, keyword));
```
Comment (ckcherry23) [inline src/main/java/seedu/address/model/person/FieldContainsKeywordsPredicate.java:58]: Could this cause a `NullPointerException` when the `jobTitle` is `null`?

Comment (ckcherry23) [reply src/main/java/seedu/address/model/person/FieldContainsKeywordsPredicate.java:58]: Same for `company`

Comment (tonishka) [reply src/main/java/seedu/address/model/person/FieldContainsKeywordsPredicate.java:58]: Thanks for pointing out, will resolve

**Review by ckcherry23 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**